blaze/expr/strings.py

```diff
@@ -54,7 +55,14 @@ def isstring(ds):
     return isinstance(getattr(measure, 'ty', measure), String)
 
 
+_add, _radd = _mkbin('add', Concat)
+_mod, _rmod = _mkbin('mod', Interp)
+_mul, _rmul = _mkbin('mul', Repeat)
+
+
 schema_method_list.extend([
+    (isstring, set([_add, _radd, _mod, _rmod, _mul, _rmul,
```

Comment:
Or, even better: put up the code you expect to work and I'll diagnose the problem.